### PR TITLE
Fixes errors related to #378

### PR DESCRIPTION
**Pull request by seemethere**

### CODE REVIEW

File: sanic/exceptions.py [hunk 1]
```diff
@@ -154,87 +155,3 @@ def __init__(self, message, content_range):
 
 class InvalidRangeType(ContentRangeError):
     pass
-  
-  
-class Handler:
```
Comment (r0fls) [inline sanic/exceptions.py]: Where does all of this code go?

Comment (seemethere) [inline sanic/exceptions.py]: It's in `handlers.py` now

**Comment by seemethere** [comment]
Fixes errors related to #378

File: sanic/exceptions.py (hunk 1 above)
Comment (seemethere) [inline sanic/exceptions.py]: The name also changed as well but I don't think too many people were subclassing it but we can add back a soft-link if people need it for legacy stuff

**Pull request merged**